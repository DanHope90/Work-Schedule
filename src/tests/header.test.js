import React from "react"; 
import { render } from "@testing-library/react"
import HeaderComponent from "../components/headerComponent";

describe("HeaderComponent", () => {
    it("render the header correctly", () => {
        const { asFragment } = render(<HeaderComponent />);
        expect(asFragment()).toMatchSnapshot();
    })
})