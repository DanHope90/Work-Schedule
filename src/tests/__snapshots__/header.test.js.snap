// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent render the header correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h1>
      Work Schedule
    </h1>
    <p>
      A simple calender to schedule your workday
    </p>
    May 27th 2022, 3:12:53 pm
  </div>
</DocumentFragment>
`;
